MapboxNavigation/ExitView.swift

```diff
@@ -141,4 +141,13 @@ class ExitView: StylableView {
         let labelTrailing = trailingAnchor.constraint(equalTo: exitNumberLabel.trailingAnchor, constant: 8)
         return [imageLeading, imageLabelSpacing, labelTrailing]
     }
+    
+    /**
+     `criticalHash(side:dataSource:)` generates the cache key needed to hold the `ExitView`'s `imageRepresentation` in the `ImageCache` caching engine.
```

Comment:
Nit: Possessive `its'` is with the apostrophe at the end.

Reply:
<grammar police>

> Nit: Possessive its' is with the apostrophe at the end.

wait, what? where?

"its" is the possessive form (think "his" or "hers" or "theirs")
"it's" is the contraction for "it is"

There is no such thing as its'

</grammar police>